<launch>

  <node pkg="move_base" type="move_base" respawn="false" name="move_base_node" output="screen" >
    
    <remap from="/base_scan" to="/Jido/Sick"/>
    <remap from="/cmd_vel" to="/Jido/Motion_Controller"/>
    <param name="footprint_padding" value="0.01" />
    <param name="controller_frequency" value="5.0" />
    <param name="controller_patience" value="100.0" /> 
    <param name="planner_frequency" value="0.5" /> 
    <param name="base_local_planner" value="waypoint_follower/WaypointFollowerPlugin" /> 
  </node>
</launch>
